Tests/ScipioKitTests/PartialCacheTests.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import Testing
+@testable import ScipioKit
+import Logging
+
+private let fixturePath = URL(fileURLWithPath: #filePath)
+    .deletingLastPathComponent()
+    .appendingPathComponent("Resources")
+    .appendingPathComponent("Fixtures")
+
+struct PartialCacheTests {
```

Comment:
@S-Shimotori I found the cause of CI failing.

Please add `@Suite(.serialized)` on your test.
And cherry-pick this commit. https://github.com/giginet/Scipio/pull/185/commits/d9415621bb2c253c65310f142d501015a49fb08f

Reply:
Thank you! I cherry-picked the commit.

Reply:
```suggestion
@Suite(.serialized)
struct PartialCacheTests {
```

Please add the same annotation here.

Reply:
I forgot that. Sorry https://github.com/giginet/Scipio/pull/184/commits/70cecbed738ce45af71ed24f82bdc563087ad903